Cargo.toml

```diff
@@ -55,6 +55,7 @@ indexmap = { version = "2", features = ["std"] }
 itoa = "1.0.6"
 itoap = { version = "1", features = ["simd"] }
 memchr = "2.6"
+memmap = { package = "memmap2", version = "0.7" }
```

Comment:
We should put this on the workspace. Then `polars-io` should also take it from there.